spimdisasm/mips/FilesHandlers.py

```diff
@@ -66,7 +66,7 @@ def getRdataAndLateRodataForFunctionFromSection(func: symbols.SymbolFunction, ro
             continue
 
         # We only care for rodata that's used once
-        if rodataSym.contextSym.referenceCounter != 1:
+        if rodataSym.contextSym.referenceCounter != 1 and len(rodataSym.contextSym.referenceFunctions) != 1:
```

Comment:
I guess for now we could add an extra `!= IDO` check meanwhile we test if this doesn't break rodata migrated on IDO projects

Reply:
Added